src/commands/index.ts

```diff
@@ -13,6 +23,9 @@ export type Command = {
   data: RESTPostAPIApplicationCommandsJSONBody;
   // Handler.
   call: (interaction: CommandInteraction) => Promise<void>;
+  // Required roles to use this command.
+  // Ignored unless `data.default_permission` is `false`.
+  allowedRoles?: string[];
```

Comment:
@DonaldKellett Is `allowedRoles` clear to you? I was first thinking of naming this `requiredRoles`, but `requiedRoles = [a, b]` looks like `a AND b`. Any suggestions?

Reply:
I didn't really pay attention to the name the last time round, but now that you mentioned it, would it be better to call it `authorizedRoles`?

Reply:
Thanks, renamed.
By the way, feel free to ask any questions and make suggestions to my PRs. Don't assume I know better. Another set of eyes can help a lot.